### ISSUE TITLE
Unable to create Wireless interface: Parent Interface Required

### ISSUE THREAD

**Issue by ghostersk** [opnsense/core#5955]
**Important notices**

Before you add a new report, we ask you kindly to acknowledge the following:

- [x ] I have read the contributing guide lines at https://github.com/opnsense/core/blob/master/CONTRIBUTING.md
- [ x] I am convinced that my issue is new after having checked both open and closed issues at https://github.com/opnsense/core/issues?q=is%3Aissue

**Describe the bug**
I am running 	OPNsense 22.7.1-amd64 FreeBSD 13.1-RELEASE OpenSSL 1.1.1q 5 Jul 2022 (default installation) on Proxmox 7.2, with passthrough Wifi card to OPNsense.
I cannot see any WIFI card in `ifconfig` but I know that it is detected with system, when I try to assign it to interface in GUI it tells me:
![Screenshot_20220813_123254](https://user-images.githubusercontent.com/68815071/184489791-9f1883e2-0089-46d2-a410-5ef4eb4ba89e.png)

**To Reproduce**

1. Interfaces: Wireless: Devices -> add new device (fails here when confirming)

**Describe alternatives you considered**

I also tried to create wlan0 with ifconfig command, what then appeared in ifconfig  but when i tried to assigne the wlan0 in OPNsense GUI to wireless interface same error
**Additional context**

```
# uname -a
FreeBSD OPNsense.test.com 13.1-RELEASE FreeBSD 13.1-RELEASE stable/22.7-n250212-a26d6065f1f SMP amd64

# sysctl net.wlan.devices
net.wlan.devices: iwm0

# pciconf -lv
iwm0@pci0:1:0:0:        class=0x028000 rev=0x10 hdr=0x00 vendor=0x8086 device=0x24fb subvendor=0x8086 subdevice=0x2110
    vendor     = 'Intel Corporation'
    device     = 'Dual Band Wireless-AC 3168NGW [Stone Peak]'
    class      = network

# dmesg | grep iwm0
iwm0: <Intel(R) Dual Band Wireless AC 3168> mem 0xfe800000-0xfe801fff irq 16 at device 0.0 on pci1
iwm0: hw rev 0x220, fw ver 22.361476.0, address 14:18:c3:06:df:4c
iwm0: <Intel(R) Dual Band Wireless AC 3168> mem 0xfe800000-0xfe801fff irq 16 at device 0.0 on pci1
iwm0: hw rev 0x220, fw ver 22.361476.0, address 14:18:c3:06:df:4c
iwm0: <Intel(R) Dual Band Wireless AC 3168> mem 0xfe800000-0xfe801fff irq 16 at device 0.0 on pci1
iwm0: hw rev 0x220, fw ver 22.361476.0, address 14:18:c3:06:df:4c
iwm0: <Intel(R) Dual Band Wireless AC 3168> mem 0xfe800000-0xfe801fff irq 16 at device 0.0 on pci1
iwm0: hw rev 0x220, fw ver 22.361476.0, address 14:18:c3:06:df:4c
iwm0: <Intel(R) Dual Band Wireless AC 3168> mem 0xfe800000-0xfe801fff irq 16 at device 0.0 on pci1
iwm0: hw rev 0x220, fw ver 22.361476.0, address 14:18:c3:06:df:4c
iwm0: <Intel(R) Dual Band Wireless AC 3168> mem 0xfe800000-0xfe801fff irq 16 at device 0.0 on pci1
iwm0: hw rev 0x220, fw ver 22.361476.0, address 14:18:c3:06:df:4c


# nano /etc/rc.conf 
legal.intel.license_ack=1
if_iwm_load="YES"
iwm3168fw_load="YES"
wlan_wep_load="YES"
wlan_ccmp_load="YES"
wlan_tkip_load="YES"

# nano /boot/loader.conf.local
if_iwm_load="YES"
iwm3168fw_load="YES"
legal.intel_iwi.license_ack=1
if_ath_load="YES"
wlan_wep_load="YES"
wlan_ccmp_load="YES"
wlan_tkip_load="YES"

# kldstat
Id Refs Address                Size Name
 1   95 0xffffffff80200000  215db98 kernel
 2    1 0xffffffff8235e000   5b9378 zfs.ko
 3    1 0xffffffff82918000     4b58 if_enc.ko
 4    1 0xffffffff8291d000     3b18 pflog.ko
 5    3 0xffffffff82921000    741a8 pf.ko
 6    1 0xffffffff82996000    2c858 if_iwm.ko
 7    1 0xffffffff829c3000     35d0 if_ath.ko
 8    2 0xffffffff829c7000     2df8 ath_dfs.ko
 9    2 0xffffffff829ca000     6478 ath_rate.ko
10    2 0xffffffff829d1000    70108 ath_hal_ar9300.ko
11    2 0xffffffff82a42000    40e70 ath_hal_ar5416.ko
12    3 0xffffffff82a83000    2d400 ath_hal_ar5212.ko
13    2 0xffffffff82ab1000    12008 ath_hal_ar5211.ko
14    2 0xffffffff82ac4000     d038 ath_hal_ar5210.ko
15    1 0xffffffff82ad2000     ba48 if_gre.ko
16    1 0xffffffff82ade000     e4d0 if_bridge.ko
17    2 0xffffffff82aed000     7870 bridgestp.ko
18    1 0xffffffff82af5000     f460 carp.ko
19    1 0xffffffff82b05000     ab48 opensolaris.ko
20    1 0xffffffff82b11000     7f30 virtio_console.ko
21    1 0xffffffff82b19000    181d0 if_lagg.ko
22    2 0xffffffff82b32000     3538 if_infiniband.ko
23    1 0xffffffff82b36000     e318 pfsync.ko
24    1 0xffffffff82b45000    fda80 iwm3168fw.ko
25    1 0xffffffff83118000     3250 ichsmb.ko
26    1 0xffffffff8311c000     2180 smbus.ko
27    1 0xffffffff8311f000    16308 qat.ko
28    1 0xffffffff83136000     20f0 coretemp.ko
29    1 0xffffffff83139000     39c0 ng_socket.ko
30    4 0xffffffff8313d000     aac8 netgraph.ko
31    1 0xffffffff83148000     31c8 ng_ether.ko
32    1 0xffffffff8314c000     53f8 ng_netflow.ko
33    1 0xffffffff83152000     31e8 ng_ksocket.ko
```

**Expected behavior**
I want to use the wireless card as AP.

**Comment by AdSchellevis** [comment]
looks like a regression in https://github.com/opnsense/core/commit/3ef64ab8ea09d2f0aea801578e985499b8e0d772, should be fixed with https://github.com/opnsense/core/commit/c11df998cb9ee1ae61b45e326d150b4889dd17d5

Try locally via a console with:
```
opnsense-patch c11df99
```

**Comment by ghostersk** [comment]
> looks like a regression in [3ef64ab](https://github.com/opnsense/core/commit/3ef64ab8ea09d2f0aea801578e985499b8e0d772), should be fixed with [c11df99](https://github.com/opnsense/core/commit/c11df998cb9ee1ae61b45e326d150b4889dd17d5)
> 
> Try locally via a console with:
> 
> ```
> opnsense-patch c11df99
> ```

Amazing Thank you!
I even ordered new Wifi card... But maybe it will be even better with new card, if FreeBSD should be better supported for that card.
